src/models/util/constants.ts

```diff
@@ -18,6 +18,7 @@ export enum DIRECTIONS {
 export enum MARCH_TYPES {
   HS = "HS",
   MINI_MILITARY = "MM",
+  SH = "SH",
```

Comment:
What is "SH"?

Reply:
Show-High step, I just realized we'd forgotten to put it in.